lib/dbapi2.py

```diff
@@ -60,6 +62,8 @@ def TimestampFromTicks(ticks):
 
 version_info = tuple([int(x) for x in version.split(".")])
 sqlite_version_info = tuple([int(x) for x in sqlite_version.split(".")])
+libsqlcipher_version = tuple([int(x) for x in re.findall(
```

Comment:
This is production code, not just test code, right? I'm not going to accept changes in production code just for the sake of tests. Especially complicated code that involves running shell commands and using regexes.